admin_commands/tremove.js

```diff
@@ -0,0 +1,30 @@
+// tremove.js
+// Clears topic at input index.
+module.exports.description = "Clears topic at the index given."
+
+const topic0p = require("../lib/topic.js");
+
+module.exports.run = (client, message, args) => {
+	const channel = message.channel;
+
+	inactiveIndexes = args.split(/\|/g);
+	inactiveIndexes = inactiveIndexes.map(s => parseInt(s.trim()));
```

Comment:
Chain the functions and use `const`. Let's also just delimit with space instead of | since it's just easier for the user to type space. Update the help message as well. Other than that, the PR is ready for merging.

Reply:
the developer copied off of you for this one